src/Illuminate/Console/Scheduling/Schedule.php

```diff
@@ -17,9 +18,14 @@
 use Illuminate\Support\Traits\Macroable;
 use RuntimeException;
 
+/**
+ * @mixin ScheduleAttributes
```

Comment:
This should be the full namespace:

```suggestion
 * @mixin \Illuminate\Console\Scheduling\ScheduleAttributes
```